.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -645,6 +646,9 @@ stages:
         debian_net5_0:
           publishTargetFramework: net5.0
           baseImage: debian
+        debian_net6_0:
+          publishTargetFramework: net6.0
+          baseImage: debian
```

Comment:
I probably should extract this to a generator step like we have for some of the other Jobs